apps/ledger-live-desktop/src/renderer/families/ethereum/StakeFlowModal/v2/StakingIcon.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { CryptoIcon, Icon, ProviderIcon } from "@ledgerhq/react-ui";
+import styled from "styled-components";
+
+type Props = {
+  icon: string;
+};
+
+const IconContainer = styled.div(
+  ({ theme }) => `
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: ${theme.space[6]}px;
+  height: ${theme.space[6]}px;
+  border-radius: 100%;
+  background-color: ${theme.colors.opacityDefault.c05};
+  margin-top: ${theme.space[3]}px;
+`,
+);
+
+const ProviderIconContainer = styled.div(
+  ({ theme }) => `
+  margin-top: ${theme.space[3]}px;
+`,
+);
+
+export function StakingIcon({ icon }: Props) {
+  if (!icon) return null;
+
+  const [iconType, iconName] = icon.split(":");
```

Comment:
In FF have it like `icon: name:type`

```suggestion
  const [iconName, iconType] = icon.split(":");
```

Reply:
Make more global